Formula/c/cataclysm.rb

```diff
@@ -8,8 +8,9 @@ class Cataclysm < Formula
 
   livecheck do
     url :stable
-    regex(/([^"' >]+)/i)
-    strategy :github_latest
+    strategy :github_latest do |json|
+      json["tag_name"]&.sub(/-RELEASE$/, "")
+    end
```

Comment:
```suggestion
    regex(/^v?(\d+(?:\.(?:\d+|\w))+(?:[_-]\d+)?)/i)
    strategy :github_latest
```

We can address this by updating the regex to explicitly match the expected version format while omitting the suffix. This is naturally more fragile than the very loose regex that this is currently using (which basically matches anything in this context) but we can always update it if/when it breaks in the future, like all the other regexes.

Reply:
agree. 👍